tools/reg_parse.c

```diff
@@ -1083,7 +1083,7 @@ check_dimspecs()
 		p->assoc_nl_var_e,p->name ) ;
 	return(1) ;
       }
-      if ( ! q->node_kind & RCONFIG )
+      if ( (! q->node_kind) & RCONFIG )
```

Comment:
```suggestion
      if ( ! (q->node_kind & RCONFIG) )
```
or
```suggestion
      if ( (~ q->node_kind) & RCONFIG )
```
I suspect `RCONFIG != 1`, so this conditional will never be met as-is.